bin/marathon-framework

```diff
@@ -72,7 +72,13 @@ function load_options_and_log {
     cmd+=( "$env_op" )
   done
   echo "${cmd[@]}"
-  logged marathon "${cmd[@]}" "$@"
+
+  if [[ "${cmd[@]} $@" == *'--no-logger'* ]]
+  then
+    marathon "${cmd[@]}" "$@"
```

Comment:
Probably you missed that, I'm not removing ```logged marathon```, I'm just adding option ```--no-logger``` so marathon can be run without logger.  
like in line 75:  function ```logged marathon``` is executed without $PATH
same way you can execute it without logged function (which just do a STDERR and STDOUT redirections) 

So the idea is that you can make logging via logger optional.